### PR TITLE
Bug fix: getRollups API, rollup jobs landing page

### PR DESCRIPTION
**Pull request by leeyun-amzn**
*Issue #, if available:* N/A

*Description of changes:*

- getRollups API fix: modify format of node API to take optional parameters.
- Modify rollup jobs landing page to show fields correctly and have working sorting function. 
- Clean up code

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by leeyun-amzn** [comment]
Output after running `yarn run test:jest`
```
Test Suites: 32 passed, 32 total
Tests:       167 passed, 167 total
Snapshots:   28 passed, 28 total
Time:        156 s, estimated 383 s
Ran all test suites.
✨  Done in 158.03s.
```

**Review by bowenlan-amzn — DISMISSED** [review]

**Comment by leeyun-amzn** [comment]
Unit tests and E2E cypress test passed in previous workflow run: https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin/runs/1772682858

**Review by bowenlan-amzn — APPROVED** [review]

File: server/services/RollupService.ts [hunk 1]
```diff
@@ -280,11 +280,18 @@ export default class RollupService {
         sortField: string;
       };
 
+      const rollupSortMap: { [key: string]: string } = {
+        _id: "rollup.rollup_id.keyword",
+        "rollup.source_index": "rollup.source_index.keyword",
+        "rollup.target_index": "rollup.target_index.keyword",
+        "rollup.rollup.enabled": "rollup.enabled",
```
Comment (thalurur) [inline server/services/RollupService.ts:287]: is `rollup.rollup.enabled` being used anywhere? don't see any occurrences in the diff

Comment (leeyun-amzn) [reply server/services/RollupService.ts:287]: This is used in `Rollups.tsx` when user selects to sort by "enabled". This `const rollupSortMap` is a way that we can quickly parse the chosen field to sort by, and send it as a parameter when calling `getRollups` API.

Comment (leeyun-amzn) [reply server/services/RollupService.ts:287]: Link to usage of `rollup.rollup.enabled`: https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin/blob/dab879565a54e83a43f03480f6143c0c3c1ab08b/public/pages/Rollups/containers/Rollups/Rollups.tsx#L363

**Review by thalurur — APPROVED** [review]

**Pull request merged**